subprojects/android-test/ui-testing-core-app/build.gradle.kts

```diff
@@ -31,6 +31,16 @@ android {
 
     testOptions {
         execution = "ANDROIDX_TEST_ORCHESTRATOR"
+
+        unitTests {
+            isIncludeAndroidResources = true
+
+            all {
+                it.systemProperty("android.junit.runner", "com.avito.android.test.app.core.AppRobolectricTestRunner")
+                it.systemProperty("robolectric.logging", "stdout")
```

Comment:
do we really need to spam in stdout?

Reply:
Tests run in a separate process, so you won't see the output in the build log.

Reply:
Why it's needed than? )

Reply:
Thank you for explanation!